docs/content/getting-started.mdx

```diff
@@ -1,98 +1,126 @@
-# Getting Started with Dagster
+# Welcome to Dagster!
 
 <p className="text-2xl mt-0 text-gray-500 tracking-tight font-light">
   Dagster is the data orchestration platform built for productivity.
 </p>
 
-## Installing Dagster
+Get started with Dagster in just three quick steps:
+
+1. [Install Dagster](#step-1-install-dagster)
+2. [Define an asset](#step-2-define-an-asset)
+3. [Materialize the asset](#step-3-materialize-the-asset)
+
+---
+
+## Step 1: Install Dagster
+
+<Note>
+  Dagster requires Python 3.6+. Refer to the{" "}
+  <a href="/getting-started/install">Installation documentation</a> for more
+  info.
+</Note>
 
 To install Dagster into an existing Python environment, run:
 
 ```bash
 pip install dagster
 ```
 
-This will install the latest stable version of the core Dagster packages in your current Python environment.
-
-## Writing a Job
+This installs the latest stable version of the core Dagster packages in your current Python environment.
 
-Let's get your first job up and running.
+---
 
-```python file=/getting_started/hello_world.py startafter=start_pipeline_marker endbefore=end_pipeline_marker
-from dagster import job, op
+## Step 2: Define an asset
 
+To get started, we'll define a simple data [asset](/concepts/assets/software-defined-assets) that represents a CSV dataset about breakfast cereals.
 
-@op
-def get_name():
-    return "dagster"
+Copy the following code. In the directory where Dagster is installed, save this into a file named `cereal.py`:
 
+```python file=/guides/dagster/asset_tutorial/cereal.py startafter=start_asset_marker endbefore=end_asset_marker
+import csv
+import requests
+from dagster import asset
 
-@op
-def hello(name: str):
-    print(f"Hello, {name}!")
 
+@asset
```

Comment:
The prior example had a dependency between two ops, and I think demonstrating that Dagster models dependencies is worth preserving.

We could use the example from the second page of the tutorial for this? https://docs.dagster.io/guides/dagster/asset-tutorial/asset-graph#lets-get-serial

Reply:
agreed. having a single asset here could confuse new users of why AssetGroup is needed at all. adding an additional downstream asset sounds great to me